cli/command/stack/deploy_composefile_test.go

```diff
@@ -0,0 +1,26 @@
+package stack
+
+import (
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/docker/pkg/testutil/tempfile"
+	"os"
+	"path/filepath"
```

Comment:
I don't think goimports understands our "two sections" style of imports, I wish it did, because I don't think this is something we should have to do manually. It just sorts them alphabetically. I have goimports run as a pre-commit hook.